EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,12 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76928
+beef1999.blogspot.com#?##banner_app
+beef1999.blogspot.com#?#.popup
+beef1999.blogspot.com#?##boz_av_bottom
+! https://github.com/AdguardTeam/AdguardFilters/issues/77063
```

Comment:
What is it?

Reply:
What? Why is it added? Lemme remove it.

Reply:
Bad merge. This rule was removed
https://github.com/AdguardTeam/AdguardFilters/commit/695921d4ba6543c4345bbc2c096880397f85ad07